components/navbar.tsx

```diff
@@ -45,7 +46,12 @@ const NavBar = (_: React.PropsWithChildren) => {
                   alt="Tzsafe logo"
                   className="h-16 w-auto"
                 />
-                <span className="ml-4 text-xs">BETA</span>
+                <div>
+                  <p className="ml-4 text-xs">BETA</p>
+                  {PREFERED_NETWORK === NetworkType.GHOSTNET && (
+                    <p className="ml-4 mt-1 text-xs">Ghostnet</p>
```

Comment:
what do you think move to under the logo? it's a bit wired on mobile.
![image](https://github.com/marigold-dev/tzsafe-ui/assets/115862/b414b186-a8db-488d-b875-706dd895200e)